webview/src/plots/components/comparisonTable/cell/ComparisonTableMultiCell.tsx

```diff
@@ -20,6 +27,13 @@ export const ComparisonTableMultiCell: React.FC<{
     dispatch(changeDisabledDragIds([]))
   }, [dispatch])
 
+  useEffect(() => {
+    window.clearTimeout(changeDebounceTimer.current)
+    changeDebounceTimer.current = window.setTimeout(() => {
+      setComparisonMultiPlotValue(path, plot.id, currentStep)
```

Comment:
Originally, I wanted to rely solely on `values` and remove `useState` but that
* was less performant (took a sec for slider change to update the image and only when the user let go of their mouse)
* made it more difficult to update the slider since the browser was handling the state instead of react